data/ga/legislature/Rick-Williams-5c79a66d-a3f7-40ce-8114-52224ce6f07a.yml

```diff
@@ -0,0 +1,35 @@
+id: ocd-person/5c79a66d-a3f7-40ce-8114-52224ce6f07a
+name: Rick Williams
+given_name: Rick
+family_name: Williams
+email: rick.williams@senate.ga.gov
+image: https://www.legis.ga.gov/api/images/default-source/portraits/williams-rick-4905.jpg
+party:
+- name: Republican
+roles:
```

Comment:
@sroomf this person served in house district 145 from January 2017 until 01/09/2023. So they may have another file somewhere.